docs/source/configuration/overview.mdx

```diff
@@ -836,6 +836,24 @@ In versions of the Apollo Router prior to 1.17, this limit was defined via the c
 
 </Note>
 
+### Early cancel
+
+Up until [Apollo Router 1.43.1](https://github.com/apollographql/router/releases/tag/v1.43.1), when the client closed the connection without waiting for the response, the entire request was canceled and did not go through the entire pipeline. Since this causes issues with request monitoring, the Router introduced a new behaviour in 1.43.1 where the entire pipeline is executed if the request is detected as canceled, but subgraph requests are not actually done. The response will be reported with the `499` status code, but not actually sent to the client.
+To go back to the previous behaviour of immediately canceling the request, the following configuration can be used:
```

Comment:
```suggestion
Up until [Apollo Router 1.43.1](https://github.com/apollographql/router/releases/tag/v1.43.1), when the client closed the connection without waiting for the response, the entire request was cancelled and did not go through the entire pipeline. Since this causes issues with request monitoring, the Router introduced a new behaviour in 1.43.1. Now, the entire pipeline is executed if the request is detected as cancelled, but subgraph requests are not actually done. The response will be reported with the `499` status code, but not actually sent to the client.
To go back to the previous behaviour of immediately cancelling the request, the following configuration can be used:
```